packages/flash-swap/test/shared/types.ts

```diff
@@ -22,12 +22,12 @@ export interface Contracts {
   balanceSheet: BalanceSheetV1;
   fintroller: FintrollerV1;
   hToken: GodModeHToken;
-  hifiFlashUniswapV2: HifiFlashUniswapV2;
-  hifiFlashUniswapV2Underlying: HifiFlashUniswapV2Underlying;
+  collateralFlashUniswapV2: CollateralFlashUniswapV2;
```

Comment:
@paulrberg would be great to re-order alphabetically and place collateralFlashUniswapV2 right below the balanceSheet variable.